ownCloud/Settings/MediaUploadSettingsViewController.swift

```diff
@@ -28,11 +29,41 @@ class MediaUploadSettingsViewController: StaticTableViewController {
 
 		if let userDefaults = OCAppIdentity.shared.userDefaults {
 			self.addSection(MediaExportSettingsSection(userDefaults: userDefaults))
+
+			let enterpriseAccountAvailable = OCBookmarkManager.shared.bookmarks.filter({$0.edition == .Enterprise}).count > 0
```

Comment:
Since you wrote that line, a new way to get the number of enterprise accounts has become available: via `OCLicenseEnterpriseProvider.numberOfEnterpriseAccounts`